### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by cody-plans**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
edit existing code. add code. plot graph.

## What did you learn from the changes you have made?
data exploration. fine tune models. use pre-trained model. 

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
tried baseline model. rnn and lstm. dropout. 

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
errror on retrieving the right content. edited url link. error on mismatched dimensions. fixed the vocab size =10000. 

## How were these changes tested?
train the model and explore the history of training records.

## Checklist
- [X] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by tianyi21 — APPROVED** [review]
Hi Ka Lok!

Thank you for submitting your DL assignments.

Your A2 is now complete.

Thanks,
Tianyi [LS]
